DirectEmployers/MyJobs
# [PD-3852] Analytics-metrics on incorrect reports

## What's this PR do?
Fixed incorrect metrics on analytics reports
## Screenshots (if appropriate)

## Where should the reviewer start?
Verify schema and schema dimension
## How should this be manually tested?
Intended to Apply and Reached Job Source should not be on site owner syndication (site analytics) 

Referral Apply Rate and Non-referral Apply Rate should not be on job owner syndication (job analytics)

## Any background context you want to provide?

## PR Checklist

### Separation of Concerns
